sig-security/annual-report-2021.md

```diff
@@ -0,0 +1,81 @@
+    *   We do not currently have any working groups.
+*   **When was your last monthly community-wide update? (provide link to deck and/or recording)**
+    *   Since our founding in October 2020, we haven’t given a monthly community-wide update yet. We have a maintainer track session coming up [at KubeCon EU 2021](https://kccnceu2021.sched.com/event/iE5u/).
+
+## Membership
+
+*   **Are all listed SIG leaders (chairs, tech leads, and subproject owners) active?**
+    *   Yes
+*   **How do you measure membership? By mailing list members, OWNERs, or something else?**
+    *   We are interested in membership metrics for the purposes of gauging the health and activity of our community. This leads us to two types of measures: measures of reach and measures of collaboration.
+    *   We measure reach using the number of members of our Slack channel and mailing list; these are potential future collaborators on SIG projects. As of April 8, 2021, there are 438 members of the main Slack channel and 131 members of the mailing list.
+    *   We measure collaboration by noting who regularly contributes to our community activities. We take attendance at our meetings, which helps with this. The 7 main SIG meetings between January 14, 2021 and April 8, 2021 have had an average of 12.4 documented attendees.
+    *   A definitive count of collaborating members across all media at any given moment is difficult to calculate, which we are comfortable with at this time.
+*   **How does the group measure reviewer and approver bandwidth? Do you need help in any area now? What are you doing about it?**
+    *   Our role in the Kubernetes project means that the majority of our reviewer bandwidth is used on triaging other SIGs’ PRs, KEPs, and Issues to bring them to our membership for comment. Note that this is a different concept of “reviewer” than is typical in a code-owning SIG.
+    *   We measure our reviewer bandwidth by noting whether everything brought to SIG Security’s attention is getting addressed in a timely fashion, and the feelings of the reviewers about their workload.
+    *   Much of this review is currently done by the SIG chairs, and it is starting to become a bottleneck. We are looking to improve our review bandwidth by bringing review of triaged issues to the regular SIG meetings for more open and inclusive participation. 
+*   **Is there a healthy onboarding and growth path for contributors in your SIG? What are some activities that the group does to encourage this? What programs are you participating in to grow contributors throughout the contributor ladder?**
+    *   We are onboarding and growing contributors continuously through our processes and culture. We actively practice behaviors of inclusion to help new and growing contributors feel more confident to participate. For example, we list appropriate pronouns in our meeting attendance list, and before changing subjects we pause to ask for additional comments.
```

Comment:
this is not a blocking question, just discovery - how are you doing outreach for new folks to come on board?